base/applications/mscutils/eventvwr/lang/ja-JP.rc

```diff
@@ -115,21 +115,24 @@ END
 IDD_LOGPROPERTIES_GENERAL DIALOGEX 0, 0, 252, 234
 STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
 CAPTION "一般設定"
-FONT 8, "MS Shell Dlg"
+FONT 9, "MS UI Gothic", 400, 0, 0x1
```

Comment:
Katayama, will this font-line-change give a good result? I just copy-pasted it from a neighbor dlg without testing it.

Reply:
I reverted that part, because this is how understood Katayamas reply. But maybe he wanted to say the exact opposite, that my change was good. I cannot say. Resolved.